.travis.yml

```diff
@@ -10,7 +10,7 @@ env:
   - E2E_TEST_HOME=$GOPATH/src/github.com/ava-labs/avalanche-testing/
   - COMMIT=${TRAVIS_COMMIT::8}
   - DOCKERHUB_REPO=avaplatform/avalanchego
-  - BRANCH=${TRAVIS_BRANCH}
+  - BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}
```

Comment:
Does this take the PR branch on a PR and other wise just the branch?
I thought $TRAVIS_BRANCH, in case of a PR, is the target branch...

Reply:
Yup I noticed this last week and found this explainer: https://unhashable.com/getting-the-current-branch-name-during-a-pull-request-in-travis-ci/.

Reply:
I'll update this one with `export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)`

Looks like `TRAVIS_PULL_REQUEST_BRANCH` is not reliable ?

https://github.com/travis-ci/travis-ci/issues/6652